cranelift-codegen/src/ir/entities.rs

```diff
@@ -209,6 +227,8 @@ pub enum AnyEntity {
     StackSlot(StackSlot),
     /// A Global value.
     GlobalValue(GlobalValue),
+    /// A constant.
+    Constant(Constant),
```

Comment:
I'll grant that this isn't clearly documented, but right now `AnyEntity` is only used for those entities which show up in clif IR, and it's mostly used for reporting errors where the user is expected to understand the index value. With constants, the syntax here makes the indices implicit. Do you actually need `AnyEntity::Constant`, or is this just something that looked needed?

Reply:
Just following the documentation at line 215: `A reference to any of the entities defined in this module`. Perhaps I should change this to `A reference to any of the entities defined in this module that can appear in CLIF IR`?

Reply:
...and I just checked `./test-all.sh` and I don't see any failures when I remove `Constant` from `AnyEntity`

Reply:
Yes, if you could make that doc change, that sounds good.

Reply:
Done in the rebase.